integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/AbstractIntegrationTest.java

```diff
@@ -166,6 +166,7 @@ protected void initiateDatalakeCreation(TestContext testContext) {
         testContext
                 .given(SdxInternalTestDto.class)
                     .withCloudStorage(getCloudStorageRequest(testContext))
+                    .withRuntimeVersion("7.2.17")
```

Comment:
A fixed version in AbstractIntegrationTest could affect tests for future versions above 7.2.17 runtime?

Reply:
Usually we have dynamically configured values for test. I would suggest to extract this there.
Please check in the application.yml and Runtime/OSUpgrade related parts. 
And also leave a comment there, that when the default runtime is X, we can use the default runtime version for testing.

Reply:
You can specify the runtime version in the individual test itself, you can look at my enterprise e2e tests to see how I do this.